src/PythonMigrationViewExtension/IronPythonDialog.xaml.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+using Dynamo.UI;
+
+namespace Dynamo.PythonMigration
+{
+    /// <summary>
+    /// Interaction logic for UserControl1.xaml
+    /// </summary>
+    public partial class IronPythonDialog : Window
```

Comment:
i'd suggest we add something to the name of this so we know what kind of dialog it is (info, warning, user choice ?)